[add] services to get, post and delete request

eliecerLR opened the pull request:
https://alkemy-labs.atlassian.net/browse/OT196-31

# Summary

- Created a function to request by GET method, using by header the JWT token stored in localstorage
-  Created a function to request by POST method, using by header the JWT token stored in localstorage and values provided in the client
- (aditional) To go faster, i created a method to delete just providing a link (just for use it being administrator)

### Evidence:

- testing the method whit a fake api (just for checking its all ok)
![image](https://user-images.githubusercontent.com/61069828/170878204-541ceb28-d94c-4871-9537-d6c0c8ab7bd1.png)

- results from get method
![image](https://user-images.githubusercontent.com/61069828/170878260-5625f61e-8054-4af2-b30b-ed49fa0ed9c6.png)

- headers from get method
![image](https://user-images.githubusercontent.com/61069828/170878287-92d4c715-9323-4aee-a44a-2331d3d3d90e.png)

GauntStrelok commented on src/services/RequestService.js:
```diff
@@ -0,0 +1,33 @@
+  const jwtToken = window.localStorage.getItem("token")
+
+  let response = await axios.get(link, {
+    headers: { Authorization: `Bearer ${jwtToken}` }
```
Ojo que el ticket dice "en el caso que existan" y aca estas agregando el header siempre

GauntStrelok commented on src/services/RequestService.js:
```diff
@@ -0,0 +1,33 @@
+      ...values,
+      token: jwtToken
+    }, {
+    headers: { Authorization: `Bearer ${jwtToken}` }
```
lo mismo aca

GauntStrelok reviewed (changes requested):
Cambios simples, y ademas hacer pull de main para arreglar conflictos

eliecerLR commented:
resolved conflicts and added a validation to know if exist a JWT token in localStorege

Evidence:

- added a test without a token
![image](https://user-images.githubusercontent.com/61069828/171226966-28bd026e-9a98-4543-bb4a-1aa9c24e0d3f.png)

- results from api
![image](https://user-images.githubusercontent.com/61069828/171227176-4378cf83-a988-42d0-9491-781a772f6e26.png)

- Headers (Authorization doesn't exist)
![Uploading image.png…]()

GauntStrelok commented on src/services/RequestService.js:
```diff
@@ -0,0 +1,47 @@
+  let header = null
+  
+  if(window.localStorage.getItem("token")) {
+    jwtToken = window.localStorage.getItem("token")
```
Esta bien, se te ocurre alguna mejor forma de escribir esto "window.localStorage.getItem("token")" y no repetirlo dos veces?
Porque estas asignandole valor al jwtToken dentro del if, no podrias hacerlo por fuera?

GauntStrelok commented on src/services/RequestService.js:
```diff
@@ -0,0 +1,47 @@
+      token: jwtToken
+    }, {
+    headers: header,
+  }
```
Cosa simple acerca de como acomodar el codigo, fijate que esta llave quedo medio afuera, y parece que cierra otra cosa, cuando en verdad simplemente cierra el objeto que esta headers adentro
